.gitlab-ci.yml

```diff
@@ -74,7 +74,7 @@ test:acceptance:
   script:
     - apk --update --no-cache add python3 gcc openssh make openssl-dev libffi-dev libc-dev python3-dev
     - cd tests
-    - pip3 install --upgrade -r requirements.txt
+    - pip3 install -r requirements.txt
```

Comment:
`Introduce dependabot and golang version` and... ?

Reply:
I consider this part of dependabot update; we use fixed requirements hence we don't need the upgrade flag.

Reply:
I'll add a comment in the commit message